sdk/enter.go

```diff
@@ -99,7 +99,7 @@ func (e *enter) MountAPI() error {
 		return err
 	}
 	// /dev/pts must be read-write because emerge chowns tty devices.
-	if err := bind("/dev/pts", filepath.Join(e.Chroot, "dev/pts")); err != nil {
+	if err := bind("/dev/pts", filepath.Join(e.Chroot, "dev/pts"), "gid=5"); err != nil {
```

Comment:
Huh, I'm unclear how this helps. As far as I know it isn't possible to change any mount options at the same time as performing a bind. Also since gid is an option for the devpts filesystem, if it is applied at all I would expect it to apply to the system globally which is probably going to be undesirable for the host system. An alternative would be to mount a new instance of the devpts filesystem but I don't know if that has other problems considering chroot isn't a fully isolated container.

We may be best off just updating the glibc ebuild to not raise this error, I just need to double check that the SDK doesn't actually care that much about /dev/pts.